conformant = "Partial"
notes = """
Does not detect type incompatibility between None and type[None].
Does not detect type incompatibility between None and incompatible protocol.
"""
output = """
specialtypes_none.py:21:1: \u001b[1m\u001b[31merror\u001b[39m\u001b[0m: in <module>: Function func1 was called with the wrong arguments [wrong-arg-types]

func1(type(None))  # E
\u001b[1m\u001b[31m~~~~~~~~~~~~~~~~~\u001b[39m\u001b[0m

specialtypes_none.py:41:1: \u001b[1m\u001b[31merror\u001b[39m\u001b[0m: in <module>: Function func2 was called with the wrong arguments [wrong-arg-types]

func2(None)  # E: not compatible
\u001b[1m\u001b[31m~~~~~~~~~~~\u001b[39m\u001b[0m

"""
conformance_automated = "Fail"
errors_diff = """
Line 27: Expected 1 errors
"""
